x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -462,6 +463,7 @@ public class MachineLearning extends Plugin
     public static final String PRE_V7_BASE_PATH = "/_xpack/ml/";
     public static final String DATAFEED_THREAD_POOL_NAME = NAME + "_datafeed";
     public static final String JOB_COMMS_THREAD_POOL_NAME = NAME + "_job_comms";
+    public static final String PYTORCH_COMMS_THREAD_POOL_NAME = NAME + "_pytorch_comms";
```

Comment:
It might be worth calling this "native inference" instead of "PyTorch".  If we ever created another native inference process, e.g. for Tensorflow or ONNX runtime, presumably those would need to share the same allocation logic and have their resources balanced with the PyTorch processes.  But we'd be stuck with "PyTorch" showing up in threadpool stats forever.